src/nodejs/integration/nodejs_app_with_seeker_test.go

```diff
@@ -39,7 +39,7 @@ var _ = Describe("CF NodeJS Buildpack", func() {
 		app.Disk = "512M"
 
 		app.SetEnv("BP_DEBUG", "true")
-		app.SetEnv("SEEKER_APP_ENTRY_POINT", "server.js")
+		app.SetEnv("SEEKER_APP_ENTRY_POINT", "dist/server.js")
```

Comment:
With the change you have done to the fixture, the root of the app should be `dist` and not the parent dir of `dist`. Only then will the detect phase pass.
Was there a particular reason you moved the entire app into a `dist/` directory?

Reply:
I mistakenly placed the `pacakge.json` inside the dist folder, I pulled it one level up, now it should be fine.

The reason for this PR is a customer having this folder structure. The buildpack did not work properly since it assumes the server.js file is in the root folder of the application (which apparently is not a correct assumption)